### ISSUE TITLE
Blank Voice Channel - VoiceStateUpdate packet assertion

### ISSUE THREAD

**Issue by xS91D** [JaxkDev/DiscordBot#49]
```
2021-12-03 [18:18:30] [Server thread/CRITICAL]: AssertionError: "Voice Channel '' not found in storage." (EXCEPTION) in "plugins/DiscordBot_v2_0_4.phar/src/JaxkDev/DiscordBot/Plugin/BotCommunicationHandler" at line 153

2021-12-03 [18:18:30] [Server thread/CRITICAL]: #0 plugins/DiscordBot_v2_0_4.phar/src/JaxkDev/DiscordBot/Plugin/BotCommunicationHandler(104): JaxkDev\DiscordBot\Plugin\BotCommunicationHandler->handleVoiceStateUpdate(object JaxkDev\DiscordBot\Communication\Packets\Discord\VoiceStateUpdate)
2021-12-03 [18:18:30] [Server thread/CRITICAL]: #1 plugins/DiscordBot_v2_0_4.phar/src/JaxkDev/DiscordBot/Plugin/Main(254): JaxkDev\DiscordBot\Plugin\BotCommunicationHandler->handle(object JaxkDev\DiscordBot\Communication\Packets\Discord\VoiceStateUpdate)
2021-12-03 [18:18:30] [Server thread/CRITICAL]: #2 plugins/DiscordBot_v2_0_4.phar/src/JaxkDev/DiscordBot/Plugin/Main(112): JaxkDev\DiscordBot\Plugin\Main->tick(integer 34978)
2021-12-03 [18:18:30] [Server thread/CRITICAL]: #3 pmsrc/src/pocketmine/scheduler/ClosureTask(61): JaxkDev\DiscordBot\Plugin\Main->JaxkDev\DiscordBot\Plugin\{closure}(integer 34978)
2021-12-03 [18:18:30] [Server thread/CRITICAL]: #4 pmsrc/src/pocketmine/scheduler/TaskHandler(137): pocketmine\scheduler\ClosureTask->onRun(integer 34978)
2021-12-03 [18:18:30] [Server thread/CRITICAL]: #5 pmsrc/src/pocketmine/scheduler/TaskScheduler(179): pocketmine\scheduler\TaskHandler->run(integer 34978)
2021-12-03 [18:18:30] [Server thread/CRITICAL]: #6 pmsrc/src/pocketmine/plugin/PluginManager(654): pocketmine\scheduler\TaskScheduler->mainThreadHeartbeat(integer 34978)
2021-12-03 [18:18:30] [Server thread/CRITICAL]: #7 pmsrc/src/pocketmine/Server(2389): pocketmine\plugin\PluginManager->tickSchedulers(integer 34978)
2021-12-03 [18:18:30] [Server thread/CRITICAL]: #8 pmsrc/src/pocketmine/Server(2152): pocketmine\Server->tick()
2021-12-03 [18:18:30] [Server thread/CRITICAL]: #9 pmsrc/src/pocketmine/Server(1992): pocketmine\Server->tickProcessor()
2021-12-03 [18:18:30] [Server thread/CRITICAL]: #10 pmsrc/src/pocketmine/Server(1586): pocketmine\Server->start()
2021-12-03 [18:18:30] [Server thread/CRITICAL]: #11 pmsrc/src/pocketmine/PocketMine(330): pocketmine\Server->__construct(object BaseClassLoader, object pocketmine\utils\MainLogger, string[10] /root/SDT/, string[18] /root/SDT/plugins/)
2021-12-03 [18:18:30] [Server thread/CRITICAL]: #12 pmsrc/src/pocketmine/PocketMine(360): pocketmine\server()```

**Comment by JaxkDev** [comment]
Known error this is the second time a different user has reported that assertion firing, will need looking into.

- Note to self, potentially build a debug file on crashes and just upload that instead of copy paste errors that is close to useless and pings every damn issue.

**Comment by JaxkDev** [comment]
Reproduced, If a member was in a voice channel before initial storage dump (plugin load) this would fire as it would have no record of the channel the member joined.

**Comment by JaxkDev** [comment]
Fixed in v2.1.1
